modules/auxiliary/scanner/redis/redis_server.rb

```diff
@@ -0,0 +1,44 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+  include Msf::Auxiliary::Redis
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Redis-server Scanner',
+      'Description'  => %q{
+          This module scans for Redis server. By default Redis has no auth. If auth
+        (password only) is used, it is then possible to execute a brute force attack on
+        the server. This scanner will find open or password protected Redis servers and
+        report back the server information
+      },
+      'Author'       => [ 'iallison <ian[at]team-allison.com>', 'Nixawk' ],
+      'License'      => MSF_LICENSE))
+
+    register_options([Opt::RPORT(6379)], self.class)
```

Comment:
Minor, but the second `self.class` argument is not necessary (that is the default value).  I prefer to not put these here unless the particular option belongs to a mixin, in which case the author should use the name of the mixin.